### PR TITLE
LateNight: fix screwed library (lib redesign)

### PR DESCRIPTION
**Pull request by ronso0**

### CODE REVIEW

File: res/skins/LateNight/style.qss [hunk 1]
```diff
@@ -1503,9 +1499,7 @@ WLibraryBreadCrumb QToolButton {
 
 WBaseLibrary QLabel {
   padding: 0px 0px 0px 0px;
-  margin: 100px 100px 100px 100px;
```
Comment (ronso0) [inline res/skins/LateNight/style.qss:1506]: that was the issue

File: res/skins/LateNight/library.xml [hunk 2]
```diff
@@ -45,14 +44,7 @@
...
                       <!--Library Tree View-->
-                      <WidgetGroup>
-                        <ObjectName>SidebarContainer</ObjectName>
-                        <Size>1min,50min</Size>
-                        <Layout>vertical</Layout>
-                        <Children>
-                          <LibrarySidebar></LibrarySidebar>
-                        </Children>
-                      </WidgetGroup>
+                      <LibrarySidebar></LibrarySidebar>
```
Comment (ronso0) [inline res/skins/LateNight/library.xml:47]: removed unnecessary nesting

File: res/skins/LateNight/library.xml [hunk 3]
```diff
@@ -67,17 +59,13 @@
...
-                  </Library>
-                </Children>
-              </WidgetGroup>
+              
+              <!-- Library Table-->
+              <Library>
+                <BgColor>#585858</BgColor>
+                <FgColor>#eece33</FgColor>
+              </Library>
+
```
Comment (ronso0) [inline res/skins/LateNight/library.xml:68]: ..here as well

**Comment by gramanas** [comment]
I'll test it soon! Thanks

**Pull request merged**